.cypress/integration/7_app_analytics.spec.js

```diff
@@ -325,19 +330,15 @@ describe('Viewing application', () => {
 
   it('Saves visualization #1 to panel', () => {
     cy.get('[data-test-subj="app-analytics-panelTab"]').click();
-    cy.get('[data-test-subj="addVisualizationButton"]').first().click();
-    cy.wait(delay);
+    cy.get('[data-test-subj="addVisualizationButton"]').last().click();
```

Comment:
why changing from first to last?

Reply:
Both options perform the same, not sure why i switched it, replacing it with first as it was originally